net/gajim/Portfile

```diff
@@ -16,37 +17,73 @@ long_description    The goal of Gajim is to provide a full featured and \
 
 maintainers         nomaintainer
 
-platforms           darwin
-supported_archs     noarch
+master_sites        https://gajim.org/downloads/1.3/
 
-master_sites        http://gajim.org/downloads/${branch}/
-use_bzip2           yes
+checksums           rmd160  c3573a515ceb3d53369b4c08a734d5785c77dd0e \
+                    sha256  5141ea1c27f4d2c1936de136dc29289a5ebf41e567dda8321e29d96518717203 \
+                    size    9578053
 
-checksums           rmd160  4bab0b953a6f9a362b43232f584a2d9bf372ef59 \
-                    sha256  46d4887857455b6e104d0478735edb51569bfa384578ffb4f4da81f1649b6a32
+depends_build       port:python38 \
+                    port:py38-setuptools \
+                    port:gettext
 
-depends_build       port:intltool \
-                    port:pkgconfig \
-                    port:autoconf \
-                    port:automake \
-                    port:libtool
+depends_run         port:py38-packaging \
+                    port:py38-nbxmpp \
+                    port:py38-openssl \
+                    port:py38-keyring \
+                    port:py38-cairo \
+                    port:py38-tinycss 
 
-depends_lib         path:lib/pkgconfig/gtk+-2.0.pc:gtk2 \
-                    port:gtkspell2 \
-                    port:py27-pygtk \
-                    port:py27-docutils \
-                    port:py27-nbxmpp \
-                    port:py27-openssl \
-                    port:dbus-python27
+depends_lib         path:lib/pkgconfig/gtk+-3.0.pc:gtk3 \
+                    port:glib2 \
+                    port:libsoup \
+                    port:py38-gobject3
```

Comment:
without looking in detail, but just quickly glancing over the installation instructions on https://dev.gajim.org/gajim/gajim/-/wikis/help/Gajim-on-macOS, it seems that there are some dependencies missing?

Reply:
@reneeotten This is what I am asking above: https://github.com/macports/macports-ports/pull/14671#issuecomment-1107515347

Reply:
the list I mention has other, non-Python packages as well I think

Reply:
oh, of course the the PyPI names are listed in upstream's setup.py

Reply:
`py-nbxmpp` dependency is horribly outdated, bumping: https://github.com/macports/macports-ports/pull/14782